lib/fog/compute/terremark/models/internetservice.rb

```diff
@@ -27,7 +27,7 @@ def save
                 protocol = self.Protocol,
                 port = self.Port,
                 options = {
-                    'Enabled' => 'true',
+                    "Enabled" => "true",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.